test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -64,14 +60,12 @@
         "inference": {
             # for shipping pillow<=6.2.2 - the last available version for py2
             "py2": ['38449', '38450', '38451', '38452'],
-            # for shipping pyyaml v5.3.1 - blocked on upgrading to v5.4.1 due to dependency on awscli
-            "py3": ['39611']
+            "py3": []
         },
         "training": {
             # for shipping pillow<=6.2.2 - the last available version for py2
             "py2": ['38449', '38450', '38451', '38452'],
-            # for shipping pyyaml v5.3.1 - blocked on upgrading to v5.4.1 due to dependency on awscli
-            "py3": ['39611']
+            "py3": []
         },
         "inference-neuron":{
             # for shipping pyyaml v5.3.1 - blocked on upgrading to v5.4.1 due to dependency on awscli
```

Comment:
Please remove `'39611'` from inference-neuron and inference-eia as well.

Reply:
Already removed in #860